mfr/server/handlers/export.py

```diff
@@ -58,6 +58,13 @@ class ExportHandler(core.BaseHandler):
     async def get(self):
         """Export a file to the format specified via the associated extension library"""
 
+        # File is already in the requested format
+        if self.metadata.ext == ".{}".format(self.format):
```

Comment:
Double check and `.lower()` the case if necessary.

Reply:
Here is the result during my testing with extra `logger.info()`:

Wrong
```bash
Exported with conversion: .PDF .pdf
```

Correct
```bash
Exported with conversion: .JPG .2400x2400.jpeg
```